tracer/src/Datadog.Trace/Iast/Telemetry/ExecutedTelemetryHelper.cs

```diff
@@ -93,9 +94,15 @@ public void GenerateMetricTags(ITags tags)
                     tags.SetMetric(GetExecutedSinkTag((IastInstrumentedSinks)i), _executedSinks[i]);
                 }
             }
+
+            ResetMetrics();
         }
 
-        ResetMetrics();
+        if (_verbosityLevel <= IastMetricsVerbosityLevel.Information)
```

Comment:
It's conter intuitive that off is not the smallest value...
`internal enum IastMetricsVerbosityLevel
{
    /// <summary>
    /// Debug
    /// </summary>
    Debug = 0,

    /// <summary>
    /// The default log level
    /// </summary>
    Information = 1,

    /// <summary>
    /// Only mandatory metrics
    /// </summary>
    Mandatory = 2,

    /// <summary>
    /// Disabled
    /// </summary>
    Off = 3,
}`

Reply:
Agree, are the values specified in the RFC?

Reply:
It follows the order of the values as they are written in the RFC:
![image](https://github.com/DataDog/dd-trace-dotnet/assets/53266532/e210eda8-2953-4062-8c7d-a23c8a0a925e)

Reply:
It does not look very intuitive, agree, but I wanted to stay aligned with the values provided in the RFC. In the java code, they do the same:

![image](https://github.com/DataDog/dd-trace-dotnet/assets/53266532/729ff211-d86c-40c7-a6ac-8975af83987f)

Reply:
I think that this internal enum values were defined like that to stay aligned with DirectSubmissionLogLevel, which follows this pattern. Still I think that it is true that a reversed order is more intuitive. I will update the code.

Reply:
I disagree, I think it _is_ intuitive to be this way 😉 

This is how most logging libraries work (Microrosoft, Serilog etc)

Think of it like this - you get fewer and fewer logs the higher the enum value